### ISSUE TITLE
Support expanding multiple source directories 

### ISSUE THREAD

**Issue by domenkozar** [tfausak/cabal-gild#13]

**Comment by tfausak** [comment]
Thanks for opening this issue! Could you explain your motivation and perhaps project layout? Most Haskell projects I've seen (and worked on) have a single source directory with all the Haskell files in it. 

As a workaround, you can have multiple `exposed-modules` (or `other-modules`), like this:

``` cabal
library
  -- cabal-gild: discover foo
  exposed-modules: ...
  -- cabal-gild: discover bar
  exposed-modules: ...
```

**Comment by domenkozar** [comment]
The use case is that I have one folder with a lot of generated Haskell GRPC types and one that I develop with it `src` and `proto`.

**Comment by tfausak** [comment]
Thanks again for opening this issue! The fix from #19 has been released in [version 1.0.2.0](https://github.com/tfausak/cabal-gild/releases/tag/1.0.2.0).
